src/Components/App/App.js

```diff
@@ -25,6 +25,40 @@ function App() {
     return setFavorites([...favorites.filter(album => album.title !== toRemove)]);
   }
 
+  const determineFav = () => {
+    if (isFav) {
```

Comment:
This function should return the `FavoriteIcon` nested in the `Tooltip` component in order to toggle the label along with the icon.